impl/src/main/resources/tck-audit-cdi.xml

```diff
@@ -2227,6 +2227,28 @@
             </assertion>
         </group>
 
+        <group>
+            <text>An alternative may be given a priority for the application by placing the |@Priority| annotation on the producer field, method or resource.
+            </text>
+            <assertion id="ca">
+                <text>Test |@Priority| on a producer method.</text>
+            </assertion>
+            <assertion id="cb">
+                <text>Test |@Priority| on a producer field.</text>
+            </assertion>
+            <assertion id="cc">
+                <text>Test |@Priority| on a resource.</text>
+            </assertion>
+        </group>
+
+        <group>
+            <text>To determine the priority of a producer method or field, the |@Priority| annotation on the producer method or field is considered first.
+                If the producer method or field does not declare the |@Priority| annotation, the |@Priority| annotation declared on the bean class is used.</text>
```

Comment:
This was my original proposal, but the final text in the spec is slightly different.

Reply:
Yes, had the changes locally but forgot to push, should be fixed now.